trade-finance-manager-ui/server/routes/case/cancellation.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
```

Comment:
Is this intentional?

Reply:
Yep, this is an issue in express (fixed in v5 I believe) that the typing incorrectly expects a `(req, res ) => void` rather than `(req, res) => Promise<void>`.

As far as I know (short of updating express) there's no fix to this :(

(e.g. see https://stackoverflow.com/questions/65670264/typescript-node-express-router-and-second-argument-typescript-eslint-no-misused)

Reply:
Thanks @MarRobSoftwire , unfortunate 😞 I can also see we already have this disabling rule in many places.